osu.Game/Overlays/OverlayTabControl.cs

```diff
@@ -32,6 +33,20 @@ public override Color4 AccentColour
             }
         }
 
+        private Color4 textColour;
+
+        public Color4 TextColour
+        {
+            get => textColour;
+            set
+            {
+                textColour = value;
+
+                foreach (var i in TabContainer.Children.OfType<OverlayTabItem>())
+                    i.TextColour = value;
+            }
+        }
+
```

Comment:
I had to separate the text colour from the bar colour, but I'm not sure if this is the right way to do it.

Reply:
Ideally this shouldn't exist.

Reply:
By the way, since that's important - *why* is it necessary exactly to separate the colours? Because if I do things with `AccentColour` then it'll probably be necessary to consider how this should behave too.

Reply:
See osu-web: in header tabs, separate colours are used for the text and for the bar below it. Previous lazer implementation didn't account for that and just used `AccentColour` for both.

Plus it's not consistent, header tabs use colours from the themed pallete while, for example, the profile section tabs just use plain white.

Reply:
Actually yeah, it has a chance of working. If the `IAccentColourProvider` is injectable it can also be provided in BDL which is like 100% better since then there's one central place for overall colouring. Or even we could entirely disregard it and just use `OverlayColourProvider` exclusively directly in the overlay tabs.

Reply:
Second option sounds much easier (and actually doable in a few lines of code I think), that is if dropping `AccentColour` in its entirety despite deriving from `OsuTabControl` would be acceptable. If we went in that direction, we'd also have to make the bar below the tabs coloured in the same way, but that's a non-issue really.

Reply:
It's not a good idea to ignore the interface members. Way I see it `AccentColour` on the tab containers needs to be fixed anyway as it's broken. If we want to disregard it entirely here than some higher-order refactoring needs to be made (decouple `AccentColour` entirely by extracting another abstract tab control) but that might paradoxically create even more owrk.